src/exampleTypes/ResultTable/index.js

```diff
@@ -41,9 +41,19 @@ let ResultTable = ({
   // From Theme/Components
   let mutate = tree.mutate(path)
   // Account for all providers here (memory provider has results with no response parent)
-  let hasResults =
-    !!_.get('context.response.results.length', node) ||
-    !!_.get('context.results.length', node)
+  let resultsLength = F.cascade(
+    ['context.response.results.length', 'context.results.length'],
+    node
+  )
+  let totalRecords = F.cascade(
+    ['context.response.totalRecords', 'context.totalRecords'],
+    node
+  )
```

Comment:
we should probably factor out a `context.response || context` method (since the `response` wrapping is going away in v1 of contexture elasticsearch)

Reply:
I think I am keeping it for now, because spending results are not showing up if I remove it.

Reply:
sorry to clarify - I didn't mean to remove it, I meant to refactor it out to a utility (which I think we have in other places in this lib) so we can reduce the surface area of change when we remove the `response` wrapping.

Reply:
disregard - we can do this later because it's already in quite a few places: https://github.com/smartprocure/contexture-react/search?q=response&type=